devices/MulticastDns/Enum/DnsResourceType.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.MulticastDns.Enum
+{
+    /// <summary>
+    /// The type of a DNS Resource.
+    /// </summary>
+    public enum DnsResourceType
+    {
+        /// <summary>
+        /// DNS Resource Type A.
+        /// </summary>
+        A = 1,
+
+        /// <summary>
+        /// DNS Resource Type CNAME.
+        /// </summary>
+        CNAME = 5,
```

Comment:
here, I think it make sense to keep those names like this rather than Pascal case

Reply:
Ok, left it in place